packages/dashboard-core-plugins/src/panels/FileExplorerPanel.tsx

```diff
@@ -166,17 +166,9 @@ export class FileExplorerPanel extends React.Component<
       return;
     }
     let newName = FileUtils.getCopyFileName(file.filename);
-    const checkNewName = async (): Promise<boolean> => {
-      try {
-        await fileStorage.info(newName);
-        return true;
-      } catch (error) {
-        return false;
-      }
-    };
     // await in loop is fine here, this isn't a parallel task
-    // eslint-disable-next-line no-await-in-loop
-    while (await checkNewName()) {
+    // eslint-disable-next-line no-await-in-loop, @typescript-eslint/strict-boolean-expressions
```

Comment:
```suggestion
    // eslint-disable-next-line no-await-in-loop
```
Doesn't seem to be necessary to remove the strict-boolean-expressions warning.

Reply:
Without disabling the strict-boolean-expressions check, I get this warning:
<img width="647" alt="image" src="https://github.com/deephaven/web-client-ui/assets/57882714/65209cca-eefc-42ea-b06c-feda00b1a858">

Should I ignore it then?

Reply:
I think your TypeScript server be trippin', if you restart it (Ctrl+P, then type `> Restart TypeScript Server`), it'll fix it. If it's still causing issues just leave it.